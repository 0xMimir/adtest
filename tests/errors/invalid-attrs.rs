fn main(){

}

#[adtest::adtest(somethings)]
fn invalid_test(){

}