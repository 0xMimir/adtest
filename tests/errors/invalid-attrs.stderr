error: Invalid attribute passed only setup and cleanup are allowed
 --> tests/errors/invalid-attrs.rs:5:18
  |
5 | #[adtest::adtest(somethings)]
  |                  ^^^^^^^^^^
